hoomd/hpmc/validation/lj_spheres.py

```diff
@@ -53,7 +53,7 @@ def run_statepoint(self, Tstar, rho_star, mean_Uref, sigma_Uref, use_clusters, u
         N = len(system.particles);
 
         if use_depletants:
-            mc = hpmc.integrate.sphere(d=0.3,seed=321,implicit=True);
+            mc = hpmc.integrate.sphere(d=0.3,seed=321);
```

Comment:
Is the `use_depletants=True` test still relevant here?

Reply:
no. removed. Thanks

Reply:
there are still two tests that call `run_statepoint` with `use_depletants=True` in `lj_spheres.py`

Reply:
removed. thanks.